ui/litellm-dashboard/src/components/key_edit_view.tsx

```diff
@@ -133,6 +133,10 @@ export function KeyEditView({
         </Select>
       </Form.Item>
 
+      <Form.Item label="Expire Key" name="duration">
+        <TextInput style={{ width: "100%" }} placeholder="e.g., 30d"/>
```

Comment:
please add a tooltip showing allowed inputs eg. 1h, 1d, 1mo etc

Reply:
Done. I also added the tooltips of the reamining options to be consistent.